### ISSUE TITLE
Improvements and generalization to liftover_bedpe

### ISSUE THREAD

**Issue by mattssca** [morinlab/GAMBLR.utils#25]
Currently, this is the only option for GMABLR to lift coordinates between the two projections (grch37 and hg38). This function is heavily restricted by the dictated data type (bedpe). 

As discussed in Slack, this function should be improved to accept additional data types (e.g maf, bed, seg). Potentially, the function could also be improved to allow the liftover of strings with genomic coordinates (i.e not restricting this function to solely operate on data frames with multiple regions). 

This should be doable without the need to bundle any additional data sets since the liftover chains are already available in GAMBLR.data. 

Lastly, this function could likely also be improved in other ways, such as added flexibility for input data column names. As reported here;

> This function seems to have some quirks that could be ironed out, too. If I include any columns other than CHROM, START, END, STRAND for A and B, I get this error:
```
Error in .local(x, ...) : strand values must be in '+' '-' '*'
```

> with STRAND specified as .
> Then, if I change STRAND to + I get

```
Error in scan(file = file, what = what, sep = sep, quote = quote, dec = dec,  : 
  scan() expected 'a real', got '+' 
```

> So I can only run it with a bedpe file with the 8 columns for CHROM, START, END, and STRAND, and STRAND has to be populated with .

**Comment by Kdreval** [comment]
This is now fixed in #25 and the update is pending review. Will close issue once update is merged.

**Comment by Kdreval** [comment]
PR is merged. Closing the issue.
